cpp/include/cudf/table/row_operators.cuh

```diff
@@ -138,6 +139,52 @@ __device__ bool equality_compare(Element const lhs, Element const rhs) {
     return lhs == rhs;
 }
 
+/**---------------------------------------------------------------------------*
```

Comment:
This is actually the code change I would intend to make to have the contains scalar method use the scalar_device_view.  Included in this PR to demonstrate how I would use the proposed scalar_device_view changes.